main.rb

```diff
@@ -0,0 +1,64 @@
+require 'json'
+require 'date'
+require './classes/item'
+require './classes/genre'
+require './classes/music_album'
+require './classes/book'
+require './classes/label'
+require './classes/author'
+require './classes/game'
+require './operations/book_operations'
+require './operations/music_album_operations'
+require './operations/game_operations'
+require './operations/file_operations'
+require './operations/item_operations'
+
+items = load_items_from_json
+
+puts 'Welcome to the Catalogue of my things'
+
+def print_options
+  puts "\nOptions"
+  puts '1. List all books'
+  puts '2. List all music albums'
+  puts '3. List all games'
+  puts '4. List all genres'
+  puts '5. List all labels'
+  puts '6. List all authors'
```

Comment:
- [ ] Great job @brighton0102 Almost all your console commands work as expected. I only encountered an error when I tried to view all `Authors` by accessing option `6` but this threw an error and shut down the app. So kindly make those changes so your app can display all authors and work as expected :wink: 
- [ ] Error Message
![image](https://github.com/brighton0102/Catalog-of-my-things/assets/102699958/5e3af85f-1ada-476a-ad3a-2efff61abe21)